packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2851,6 +2851,47 @@ class ListBody extends MultiChildRenderObjectWidget {
 /// [CustomMultiChildLayout] instead. In particular, when using a [Stack] you
 /// can't position children relative to their size or the stack's own size.
 ///
+/// {@tool sample}
+///
+/// This example shows how Stack can be used to enhance text visibility
+/// by adding gradient backdrops.
+///
+/// ```dart
+/// SizedBox(
+///   height: 250,
+///   child: Stack(
+///     fit: StackFit.expand,
+///     children: <Widget>[
+///       Image.asset(
```

Comment:
This will be really complicated for consumers to play with.

Can you instead just use a background that has low contrast with your foreground text?